frontend/src/views/CanteenEditor/CanteenForm.vue

```diff
@@ -503,25 +500,23 @@ export default {
           this.$emit("updateCanteen", canteenJson)
           if (this.isNewCanteen) {
             const canteenUrlComponent = this.$store.getters.getCanteenUrlComponent(canteenJson)
+            const name = this.showSatelliteCanteensCount
```

Comment:
Ça t'irait de séparer ces deux opérations ternaires pour les rendre plus lisibles?

Reply:
You don't find this easy to read?! :laughing: